bin/run-in-env

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env ruby
+
+require 'getoptlong'
+require 'logger'
+require 'yaml'
+require 'aws-sdk-ssm'
+require 'erb'
+
+@log = Logger.new(STDOUT)
+@log.level = Logger::INFO
+
+def usage
+  puts <<~EOF
+  Usage: run_in_env.sh -e <env> [-o <output>] [-f <output_file>] [-- <command>]
+  Arguments:
+    -e (--env): environment
+    -o (--output):
+      stdout (default): Write raw config to standard out
+      env_file: Write to a .env file to set up shell environment. Must provide -f
+      subshell: Run command in a subshell with env. variables. Must provide command
+      tf_subshell: Run command in a subshell with terraform env. variables. Must provide command
+    -f (--output_file): path to output file
+  EOF
+  exit(1)
+end
+
+def should_run_command_in_subshell(output)
+  output == 'subshell' || output == 'tf_subshell'
+end
+
+def run_command_with_env(config_map, command)
+  @log.debug 'Running command: ' + command
+  ENV.update(config_map)
+  exec command
+end
+
+def pull_config_from_aws(env)
+  param_name = "/config/#{env}"
+  @log.debug('Fetching parameter ' + param_name)
+  ssm_client = Aws::SSM::Client.new(region: 'eu-west-2')
+
+  response = ssm_client.get_parameter({
+    name: param_name,
+    with_decryption: true,
+  })
+
+  YAML.load(response.parameter.value)
+end
+
+def make_tf_vars_map(config_map)
+  @log.debug 'Transforming variables into terraform TF_VAR_* variables'
+
+  config_map.map { |k, v|
```

Comment:
In ruby there is a convention that blocks spanning on one line we you use `{}`

`config_map.map { |k, v| ["TF_VAR_#{k}", v] }.to_h`

or if you want to go on multiple lines you do:

```
config_map.map do |k, v| 
  ["TF_VAR_#{k}", v]
end.to_h
```
which is not that nice considering that at the end of it you have to do `.to_h`

Reply:
done, thanks